.pre-commit-config.yaml

```
@@ -0,0 +1,9 @@
+repos:
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.41.0
```

```suggestion
    rev: v0.45.0
```

https://github.com/igorshubovych/markdownlint-cli/releases/tag/v0.45.0